deployment/base/resources-db.yaml

```diff
@@ -128,13 +127,15 @@ data:
 
     CREATE TABLE "Operating_Room_Reservation" (
       "ID_reservation" serial PRIMARY KEY,
+      "ID_plan" int,
       "ID_operating_room" int,
       "Reservation_date" date,
       "Start_time" time,
       "End_time" time
     );
 
-    ALTER TABLE "Surgical_Plans" ADD FOREIGN KEY ("ID_operating_room") REFERENCES "Operating_Room" ("ID_operating_room");
+
```

Comment:
don't we agree to delete this line too? 
https://github.com/BoredOtter/hms/blob/master/deployment/base/resources-db.yaml#L105